### ISSUE TITLE
CalendarExtender EndDate property is not in effect in Internet Explorer 8

### ISSUE THREAD

**Issue by sonergonul** [DevExpress/AjaxControlToolkit#238]
[SamplePage.txt](https://github.com/DevExpress/AjaxControlToolkit/files/506256/SamplePage.txt)

### The installation method
- [X] Installer

I installed this version on Visaul Studio 2015. And added a calendar to a blank web page. After that, I set it's `EndDate `property to today's date with

`CalendarExtender1.EndDate = DateTime.Today;`

line. It perfectly works (can't select future dates and they shown as ~~28~~ for example) on Google Chrome and Firefox. But in Internet Explorer, future dates shown as 28 (normal) they _can_ be selected.

Actual result screenshot belongs on IE and Expected result screenshot belongs on Firefox and Chrome.

### Actual result

![image](https://cloud.githubusercontent.com/assets/1696649/19037223/cf3d048c-897d-11e6-9211-30e1204fa6be.png)

### Expected result

![image](https://cloud.githubusercontent.com/assets/1696649/19037276/0fc4b39c-897e-11e6-82eb-8ba6f416533d.png)

### Browser(s) used

Google Chrome 53.0.2785.116
Firefox 48.0.2
Internet Explorer 11.0.9600.18230

### A site deployment method
- [ ] VS development web-server,
- [X] IIS

**Comment by sonergonul** [comment]
Hi @MikhailTymchukDX 

Here is my full web form and .CS file code:

[Sample.docx](https://github.com/DevExpress/AjaxControlToolkit/files/506293/Sample.docx)

**Comment by MikhailTymchukDX** [comment]
It's hard to recreate a working example from this code, so let's base our discussion on the sample project I attached: [CalendarTest.zip](https://github.com/DevExpress/AjaxControlToolkit/files/506319/CalendarTest.zip)

The AJAX Control Toolkit [NuGet package v.16.1.0](https://www.nuget.org/packages/AjaxControlToolkit/16.1.0) is used here.
Code-behind that sets `EndDate` matches your code.

Please run this project in different browsers and inform us of your results.

**Comment by sonergonul** [comment]
Interesting @MikhailTymchukDX , your code does _not_ procude the result what I get.

I see a few different things between your code and mine:

 1. I downloaded relevant .exe from https://www.devexpress.com/Products/AJAX-Control-Toolkit/ page.
 2. I used `body` and `form` html elements, not you.
 3. You used `Content`, I don't.
 4. You used `ajaxToolkit:CalendarExtender` but mine is`<asp:CalendarExtender` which I specify it on `TagPrefix="asp"`.

Do you think these things might generate the difference?

**Comment by MikhailTymchukDX** [comment]
>Interesting @MikhailTymchukDX , your code does not procude the result what I get.

Yes, I forgot to mention that my sample does not produce the result you got.
The goal is to understand the difference to find the cause of the issue.

>Do you think these things might generate the difference?

No, these differences are not relevant to our situation.
So, please specify the following:
* Your system locale
* Errors that you see in a browser's console (F12), if any

**Comment by sonergonul** [comment]
> Your system locale

I'm using Win7 Enterprise SP1 x64. My `CurrentCulture` is `tr-TR`.

> Errors that you see in a browser's console (F12), if any

I don't see _any_ error in developers tools/console.

![image](https://cloud.githubusercontent.com/assets/1696649/19064712/adb92eae-8a17-11e6-9731-6610a96970ad.png)

**Comment by MikhailTymchukDX** [comment]
@sonergonul Thank you so much for the screenshot!
It indicates that your document mode is set to emulate IE 7, which does not meet the [minimum requirements](https://github.com/DevExpress/AjaxControlToolkit/wiki#browser-support).

Would you please set the Document mode to `Edge` and check your result?
![image](https://cloud.githubusercontent.com/assets/17080984/19067951/20609166-8a29-11e6-9205-ab5625f7f86d.png)

**Comment by sonergonul** [comment]
@MarinaRukavitsyna 

Oh, that's wonderful to know. Thank you very much.

It works on 9, 10 and Edge as you mentioned. Now I searched how to provide a functionality to my webform that works always on Edge document mode and I found that can be solved using `meta ` html tag as:

`<meta http-equiv="X-UA-Compatible" content="IE=edge"/>`

By the way, maybe you might wanna add these Document Mode compabilities to your [wiki ](https://github.com/DevExpress/AjaxControlToolkit/wiki) page also?
